src/VisualStudio/Core/Def/Implementation/TableDataSource/Suppression/VisualStudioDiagnosticListSuppressionStateService.cs

```diff
@@ -31,6 +34,9 @@ internal class VisualStudioDiagnosticListSuppressionStateService : IVisualStudio
         private int _selectedCompilerDiagnosticItems;
         private int _selectedNonSuppressionStateItems;
 
+        private const string SynthesizedFxCopDiagnostic = "SynthesizedFxCopDiagnostic";
+        private readonly string[] SynthesizedFxCopDiagnosticCustomTags = new string[] { SynthesizedFxCopDiagnostic };
+
         [ImportingConstructor]
         public VisualStudioDiagnosticListSuppressionStateService(
             SVsServiceProvider serviceProvider,
```

Comment:
For now, the users of the service are required to invoke it from UI thread (e.g. new baseline command in stancore explicitly switch to UI thread for accessing the error list state/data.)